starter-code/client/src/pages/ride-files/JoinPopup.js

```diff
@@ -62,21 +117,25 @@ export default function JoinPopup({open,handleClose,rideInfo,user,loading_effect
                       Distance that the driver is willing to go to pick up riders: {rideInfo.rider_radius} meters
                     </Typography>
                   }
-                  {driver_pickup &&
-                    <TextField
-                      onChange={(e) => setAddress(e.target.value)}
-                      autoFocus
-                      margin="dense"
-                      label="Pickup Address"
-                      fullWidth
-                      variant="standard"
-                    />
-                  }
-                  {!driver_pickup &&
-                    <Typography variant="body1" gutterBottom component="div">
-                      The driver wants to meet only at their specified leave location.
-                    </Typography>
-                  }
+                  <div ref = {ref}>
+                    {driver_pickup &&
+                      <TextField
+                        value = {value}
+                        onChange={handleInput}
+                        autoFocus
+                        margin="dense"
+                        label="Pickup Address"
+                        fullWidth
+                        variant="standard"
+                      />
+                    }
+                    {!driver_pickup &&
+                      <Typography variant="body1" gutterBottom component="div">
+                        The driver wants to meet only at their specified leave location.
```

Comment:
Maybe change to "Start Location" since the drive submission form has "Start Location"